src/lib/services/environment-service.ts

```diff
@@ -92,6 +99,7 @@ export default class EnvironmentService {
     async addEnvironmentToProject(
         environment: string,
         projectId: string,
+        user = 'unknown',
```

Comment:
Does that mean that it'll show up as "unknown" for everything until we eventually patch it?

Reply:
And maybe we could be clearer here: what is the `user`? It's a string, right? But is it the username? The email? Something else? But this may also be how we do it everywhere else, so 💁🏼

Reply:
The controller calling this code will pass the actual username and populate event logs with a correct value in production but all the tests don't need to be updated because of the default value.

Reply:
It's actually username or email:
```
export function extractUsernameFromUser(user: IUser): string {
    return user?.email || user?.username || 'unknown';
}
```